commands/host.vegman

```diff
@@ -224,6 +224,55 @@ function cmd_virtualmedia_umount {
   usb-ctrl eject "usb.ms.cli"
 }
 
+# @restrict cmd_config tech,admin
+# @doc cmd_config
+# Host config backup and restore
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_default tech,admin
```

Comment:
Remove `tech` user mentioning as it's for internal using only and will be added by yocto.